src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -105,8 +105,16 @@ public function match(\$pathinfo)
     {
         \$allow = array();
         \$pathinfo = rawurldecode(\$pathinfo);
+        \$trimmedPathinfo = rtrim(\$pathinfo, '/');
         \$context = \$this->context;
         \$request = \$this->request;
+        \$requestMethod = \$isLikeGetMethod = \$context->getMethod();
+        \$schema = \$context->getScheme();
+
+        if ('HEAD' === \$requestMethod) {
+            \$isLikeGetMethod = 'GET';
```

Comment:
I like `conicalMethod`, a lot. I had previously opted to use `methodWhereHeadMatchesGet` instead. Which is a little verbose, but it does make it clear what's going on.